examples/example-gateway/build.yaml

```diff
@@ -9,3 +9,12 @@ packageRoot: github.com/uber/zanzibar/examples/example-gateway
 targetGenDir: ./build
 thriftRootDir: ./idl
 traceKey: x-trace-id
+moduleSearchPaths:
+- clients/*
+- middlewares/*
+- endpoints/*
```

Comment:
Since this is glob, I would not be surprised to see patterns like "endpoints/*/**" in the yaml file, can we have such a case to make sure things work as intended?

Reply:
`**` is not supported, see https://golang.org/pkg/path/filepath/#Match

Reply:
okay, does that mean `endpoint/*` does not match `endpoint/foo/bar/endpoint-config.yaml`?

Reply:
Yes, you'd need to write `endpoint/*/*` to capture that, we might want a more flexible globbing pattern library but I just used the standard library one for now.

Reply:
Okay, got it, this is definitely something worth to document.